integration-tests/tests/cases/chains.rs

```diff
@@ -0,0 +1,13 @@
+use integration_tests::cluster;
+
+#[test_log::test(tokio::test)]
+async fn test_solana_signature_basic() {
+    let cluster = cluster::spawn().sol().await.unwrap();
+    let _outcome = cluster
+        .sign()
+        .sol()
+        .payload([42u8; 32])
+        .path("test/integration/path")
+        .await
+        .unwrap();
+}
```

Comment:
By moving all the logic, including assertions, inside the setup, it can be hard to see what the test actually does. Please correct me if I'm getting this wrong.

From my code reading, I understand this submits a sign request to a locally emulated Solana chain and then waits for the response to arrive within 90s. If it finds a matching event, it parses the signature and returns it.

Could we add a check that the signature matches the original request payload and path?